estate/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "E-State!",
+
+    'summary': """
+        Manage real estate advertisements
+    """,
+
```

Comment:
```suggestion
    'name': 'E-State!',
    'summary': 'Manage real estate advertisements',
```

Remove useless blanks and prefer using double quotes ("") for long strings, multiline or user inputs/displays.